lib/page/scroll_page.dart

```diff
@@ -83,16 +83,16 @@ class _ScrollPageState extends State<ScrollPage> {
           },
         ),
         ),
-        Obx(()=>DrawerController(
-            key: drawerKey,
-            alignment: DrawerAlignment.end,
-            isDrawerOpen: _userPageController.showRightMenu.value,
-            drawerCallback: (isOpened){
-              _userPageController.toggleRightMenu();
-            },
-            child: UserRightMenuWidget()
-        ),
-        ),
+        // Obx(()=>DrawerController(
```

Comment:
Add `TODO:` here and describe why we comment out this. so that we can easily figure out how to add it back.